src/Makefile

```
@@ -73,4 +76,35 @@ bench: hashtree.h ubench.h bench.c $(libname)
 	$(CC) $(CFLAGS) $(LDFLAGS) -o bench bench.c $(benchlibs)
 
 clean:
-	-rm -f $(objarm) $(objx86) libhashtree.a libhashtree.lib test test.exe bench 
+	-rm -f $(objarm) $(objx86) libhashtree.a libhashtree.lib test test.exe bench hashtree.pc
+
+ifeq ($(PREFIX),)
+PREFIX := /usr
+endif
+
+hashtree.pc: .FORCE
+	@echo 'prefix='$(PREFIX) >> hashtree.pc
+	@echo 'exec_prefix=$${prefix}' >> hashtree.pc
+	@echo 'libdir=$${prefix}/lib' >> hashtree.pc
+	@echo 'includedir=$${prefix}/include' >> hashtree.pc
+	@echo '' >> hashtree.pc
+	@echo 'Name: hashtree' >> hashtree.pc
+	@echo 'Description: Fast hashing of Merkle trees' >> hashtree.pc
+	@echo 'Version: '$(VERSION) >> hashtree.pc
```

Some [tools](https://manpages.ubuntu.com/manpages/impish/man5/pc.5.html) require a URL
```suggestion
	@echo 'Version: '$(VERSION) >> hashtree.pc
	@echo 'URL: https://github.com/prysmaticlabs/hashtree' >> hashtree.pc
```